tracpro/contacts/tests/test_models.py

```diff
@@ -73,20 +73,19 @@ def test_create(self):
     def test_get_or_fetch_existing_local_contact(self, mock_kwargs_from_temba):
         self.mock_temba_client.get_contacts.return_value = TembaContact.create(
             uuid='C-007', name="Mo Polls",
-            urns=['tel:078123'], groups=['G-001', 'G-005'],
+            urns=['tel:078123'], groups=[self.group3, self.group5],
             fields={},
             language='eng', modified_on=timezone.now())
         # get locally
         contact = models.Contact.get_or_fetch(org=self.unicef, uuid='C-001')
         self.assertEqual(contact.name, "Ann")
 
-    @skip("Skipping test_get_or_fetch_non_existing_local_contact() for now, fixing functionality for API v2.")
     def test_get_or_fetch_non_existing_local_contact(self):
         mock_contact = TembaContact.create(
             name='Mo Polls',
             uuid='C-009',
             urns=['tel:123'],
-            groups=[self.region1.uuid, self.group1.uuid],
+            groups=[self.region1, self.group1],
```

Comment:
I think since this mock_contact is supposed to come from RapidPro, the idea is that groups can be, what is known to TracPro as, either a Panel (region) or a Cohort (group).  Is that more confusing or less confusing? 😄